.ci/install-yq.sh

```diff
@@ -56,12 +56,7 @@ function install_yq() {
 		die "Please install curl"
 	fi
 
-	# Workaround to get latest release from github (to not use github token).
-	# Get the redirection to latest release on github.
-	yq_latest_url=$(curl -Ls -o /dev/null -w %{url_effective} "https://${yq_pkg}/releases/latest")
-	# The redirected url should include the latest release version
-	# https://github.com/mikefarah/yq/releases/tag/<VERSION-HERE>
-	yq_version=$(basename "${yq_latest_url}")
+	local yq_version=2.3.0
```

Comment:
@grahamwhaley - can you put your @jodh-intel hat on and PTAL?

Reply:
I think this is OK, to pin, to get stability. What I don't quite follow is that the curl of github release `latest` should be picking up the latest released version (which is 2.3.0 for yq right now), and not any `HEAD` version.
Thus, I actually think:
 - this change is currently benign
 - but in the long term, pinning to a known version probably gets us more stability than not pinning - but at the cost that we have to track.
Hmm, actually then, @chavafg - do we think this `yq` version number should be wired into the `versions.yaml` files, and not hard wired here in a script??

Reply:
@egernst @grahamwhaley I agree with put the request version to versions.yaml.
But yq is used to parse versions.yaml.

Reply:
@teawater - heh, hah!
In which case, I have no issue with this - as long as we remember it is hard wired ;-)